.github/config/labeler.yml

```diff
@@ -18,4 +20,5 @@ lang:makefile: ['**/Makefile']
 lang:sql: ['**/*.prisma', '**/*.sql']
 lang:shell: ['**/*.sh', '**/*.sh.hbr', '**/*.ps1.hbr']
 lang:golang: ['**/*.go']
-lang:typescript: ['**/*.ts', '**/*.js']
+lang:typescript: ['**/*.ts', '**/*.tsx']
```

Comment:
`ts?` is not possible?

Reply:
what?

Reply:
Use of glob whitecard characters could also be a possibility. 
```
lang:typescript: ['**/*.ts?', '**/*.js?']
```

Reply:
If someone adds a random file with a .ts extension, it will be incorrect. I believe explicit types would be better in that case. What are your thoughts @nandor-magyar ?

Reply:
ah, true, tsc, or tsw, or tsa is not a valid extensions, and ? would match those as well.